Lecture7/Testing_Task1/Helper/WebDriverUtils/ChromeDriverSettings.java

```diff
@@ -0,0 +1,15 @@
+package Helper.WebDriverUtils;
+
+class ChromeDriverSettings {
```

Comment:
Didn't catch the logic of this class
You can use like fields variables in this class, but if you are returning constants so use it like constants

Reply:
fixed